src/web/app/pages-instructor/instructor-session-result-page/instructor-session-no-response-panel.component.ts

```diff
@@ -57,6 +77,52 @@ export class InstructorSessionNoResponsePanelComponent implements OnInit, OnChan
     }
   }
 
+  openSendReminderModal(event: any): void {
+    event.stopPropagation();
+
+    const courseId: string = this.session.courseId;
+    const feedbackSessionName: string = this.session.feedbackSessionName;
+
+    const modalRef: NgbModalRef = this.modalService.open(SendRemindersToStudentModalComponent);
+    modalRef.componentInstance.courseId = courseId;
+    modalRef.componentInstance.feedbackSessionName = feedbackSessionName;
+
+    forkJoin(
```

Comment:
If we sent the request at the root page, can we reuse `noResponseStudents` and don't send the request again?

https://github.com/TEAMMATES/teammates/blob/55bc31496aa06ffd31350593cc7b7258af408af0/src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-page.component.ts#L192-L213